src/hooks/autocomplete.ts

```diff
@@ -1,6 +1,6 @@
 import {useState, useRef, useEffect} from 'react';
 
-import useGoogleMap from './map-instance';
+import {useGoogleMap} from './map-instance';
 
 interface AutocompleteProps {
```

Comment:
We could export the interfaces as well. What do you think? (If so, this must be done for the other hooks as well)
```suggestion
export interface AutocompleteProps {
```

Reply:
Is okay for me, but do we need them at any other places? (just wondering)

Reply:
We don't need them necessarily, but I was just thinking about a developer, who works with this library and wants to create a props object, that he/she can pass to the hook and wants to type this object right the first time. For this case it would be nice, if it's possible to import the props interface from our library.

Reply:
Yes, that's a point :) done!